apps/web/src/app/_components/homepage/preview-card.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { Badge } from "@tremor/react";
+import { SiDiscord } from "@icons-pack/react-simple-icons";
+import { LineChart } from "@tremor/react";
+
+const data = {
+  user: "John Doe",
+  timestamp: "04/05/2024 4:07PM",
+  message: "Human Livestocks sucks",
+  score: -20.26,
+  reaction: "👎",
+  reactionCount: 12,
+  chart: [
+    {
+      date: "Apr 1",
+      "John Doe": 23.38,
+    },
+    {
+      date: "Apr 2",
+      "John Doe": 21.03,
+    },
+    {
+      date: "Apr 3",
+      "John Doe": 21.94,
+    },
+    {
+      date: "Apr 4",
+      "John Doe": 21.08,
+    },
+    {
+      date: "Apr 5",
+      "John Doe": 0.82,
+    },
+  ],
+};
+
+export default function PreviewCard() {
+  return (
+    <div className="relative ml-0 w-full max-w-lg scale-90 rounded-xl border border-tremor-border p-4 md:scale-100">
```

Comment:
wtf

Reply:
I think it'd be cleaner to just use px/rem values for the sizing of this container